.github/workflows/module-prep-release.yml

```diff
@@ -0,0 +1,128 @@
+name: Prepare Release
+# This workflow is manually dispatched. It:
+# - bumps the version as specified in the input
+# - prepares build artifacts
+# - creates a release branch
+# - commits the changes
+# - creates a pull request to the main branch
+# - TODO: consider skipping the PR, push to main, and create a release automatically
+
+on:
+  workflow_call:
+    inputs:
+      module-repo:
+        description: The module repo.
+        type: string
+        required: true
+      module-branch:
+        description: The branch of the module to use.
+        type: string
+        required: true
+      level:
+        description: The level of release to be used.
+        type: string
+        default: 'patch'
+        required: true
+
+jobs:
+  prep-release:
+    name: Prepare Release
+    runs-on: ubuntu-latest
+    permissions:
+      # Give the default token permission to commit, push the changed files, and open a pull request.
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+        with:
+          repository: ${{ inputs.module-repo }}
+          ref: ${{ inputs.module-branch }}
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@cf4cade2721270509d5b1c766ab3549210a39a2a  # v2.33.0
+        with:
+          php-version: '8.1'
+          coverage: none
+          tools: composer, cs2pr
+
+      - name: Setup Node.js
+        uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020  # v4.4.0
+        with:
+          node-version: 20.x
+          cache: 'npm'
+
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+      - name: Cache Composer vendor directory
+        uses: actions/cache@5a3ec84eff668545956fd18022155c47e93e2684  # v4.2.3
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Show versions
+        run: |
+          php --version
+          composer --version
+          node --version
+          npm --version
+
+      - name: Validate composer.json and composer.lock
+        run: composer validate
+
+      - name: Install PHP Dependencies
+        run: composer install --no-dev --optimize-autoloader
+
+      - name: Install Packages
+        run: npm install
+
+      # check set-version-bump exists
```

Comment:
This step comment says essentially the same thing as the name. Let's just merge it into the name and delete the comment.